backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -229,6 +230,9 @@ public Endpoint createEndpoint(@Context SecurityContext sec,
         }
 
         if (isForOpenBridge(endpoint)) {
+            CamelProperties cp = endpoint.getProperties(CamelProperties.class);
+            checkSlackChannel(cp);
```

Comment:
We are only supporting slack for now. We will need to refactor this call (and probably there are other slack-specific stuff around in the update).

Reply:
Yes, the entire RHOSE code in notifications is tightly-coupled with Slack. We'll refactor everything when another kind of sink will be available.